From: Alexis Murzeau <amubtdx@gmail.com>
Date: Sat, 4 Apr 2020 15:16:10 +0200
Subject: docs: use recommonmark as an extension

Sphinx has deprecated source_parsers which is used for recommonmark.

This patch replace the use of source_parsers for recommonmark with the
use of recommonmark as an extension (supported by recommonmark since
0.5.0).

Forwarded: https://github.com/streamlink/streamlink/pull/3204
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 docs/conf.py | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 7b05bb3..2c8cc44 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -18,17 +18,11 @@ sys.path.insert(0, os.path.abspath('.'))
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'ext_argparse', 'ext_github', 'ext_releaseref']
+extensions = ['sphinx.ext.autodoc', 'ext_argparse', 'ext_github', 'ext_releaseref', 'recommonmark']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
-# The suffix of source filenames.
-source_suffix = ['.rst', '.md']
-source_parsers = {
-   '.md': 'recommonmark.parser.CommonMarkParser',
-}
-
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
